### PR TITLE
Update to add MLwithTensorFlow2ed

### PR DESCRIPTION
**Pull request by chrismattmann**
Add MLwithTensorFlow2ed to your list. Thank you for your consideration!

### CODE REVIEW

**Comment by nehaaaa6** [comment]
# First, make sure you have pip installed
# Then, open your terminal or command prompt and enter:

pip install tensorflow scikit-learn

# This will install the latest versions of TensorFlow and scikit-learn
# Once the installation is complete, you can start using these libraries in your Python code:

import tensorflow as tf
from sklearn import datasets

# Example code using TensorFlow and scikit-learn
# (note: this is just a toy example, not a real-world ML model!)

# Load the iris dataset from scikit-learn
iris = datasets.load_iris()

# Split the dataset into training and testing sets
X_train, X_test, y_train, y_test = train_test_split(iris.data, iris.target, test_size=0.2, random_state=42)

# Define the TensorFlow model architecture
model = tf.keras.Sequential([
  tf.keras.layers.Dense(10, activation='relu', input_shape=(4,)),
  tf.keras.layers.Dense(3, activation='softmax')
])

# Compile the model
model.compile(optimizer='adam', loss='sparse_categorical_crossentropy', metrics=['accuracy'])

# Train the model on the training data
model.fit(X_train, y_train, epochs=50, validation_split=0.2)

# Evaluate the model on the testing data
loss, accuracy = model.evaluate(X_test, y_test)

print('Test loss:', loss)
print('Test accuracy:', accuracy)

**Comment by chrismattmann** [comment]
hi, just bumping this @jtoy thanks for your consideration.
